### PR TITLE
Modal- about section information

### LINKED ISSUE #20: Add Pokemon Cry Audio
**Is your feature request related to a problem? Please describe.**
Need pokemon cry button and audio next to pokemon name in Modal.

**Describe the solution you'd like**
Fetch audio, define function to handle click for button.

### LINKED ISSUE #35: Modal About Section information
**Is your feature request related to a problem? Please describe.**
Break up existing modal issue into two sections:
  - Implement modal with basic styling and information (previous issue)
  - Finish implementing modal with pokemon cry, title bar, and information for each section (this issue)

**Describe the solution you'd like**
Display info for each title section for pokemonDetails

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
1. Display About section information in modal.
2. Refactor parent - child state/props to handle modalData object.

Closes #35 
Closes #20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.4.0",
+    "@emotion/styled": "^11.3.0",
+    "@material-ui/core": "^5.0.0-beta.1",
```
Comment (BlackCubes) [inline package.json:8]: MaterialUI 🤯

**Review by BlackCubes — APPROVED** [review]
Dis jooicy! Good job!

**Review by Gjhnsn — APPROVED** [review]

**Review by connielion — APPROVED** [review]
Looks good.

**Comment by BlackCubes** [comment]
Beautiful

**Pull request merged**